public/js/index.js

```diff
@@ -484,9 +498,21 @@ Index.handleContextMenu = function (option, id) {
         LRR.openInNewTab(`./edit?id=${id}`);
         break;
     case "delete":
-        if (window.confirm("Are you sure you want to delete this archive?")) {
-            Server.deleteArchive(id, () => { document.location.reload(true); });
-        }
+        window.Swal.fire({
+            title: "Are you sure?",
+            text: "This is a destructive operation! Are you sure you want to delete this archive?",
+            icon: "warning",
+            showCancelButton: true,
+            focusConfirm: false,
+            confirmButtonText: "Yes, delete it!",
+            reverseButtons: true,
+            confirmButtonColor: "#d33",
+            cancelButtonColor: "#3085d6",
```

Comment:
I believe non-destructive buttons should follow the color of the theme's buttons. 
```suggestion
            cancelButtonColor: $('.stdbtn').css("backgroundColor"),
```
If there's a way to attach a class to buttons, it'd probably be easier to add the `stdbtn` class to them here.

Reply:
I prefer use the original color.
```js
confirmButtonColor: "#d33",
```
1. ![image](https://user-images.githubusercontent.com/10560038/166283055-b681ce5a-4bbf-41d6-9934-cfc71a98609c.png)
![image](https://user-images.githubusercontent.com/10560038/166287880-55807bf8-2766-4dcf-b6b5-eca4f992b71b.png)


True that we can use 
```js
customClass: {
    cancelButton: "stdbtn",
},
confirmButtonColor: "#d33",
```
to attach stdbtn class to the button.
2. ![image](https://user-images.githubusercontent.com/10560038/166281386-ce102fb8-f781-40e1-b4dd-042289ce953f.png)
![image](https://user-images.githubusercontent.com/10560038/166287546-65d67303-e8aa-4a39-9b85-2c41517375e0.png)

However, as you can see, the existing css only have the `button`, and don't have `cancelButton`. Also, the shape, on:hover and formating is different than the SweetAlert one's. So you we may rather write new classes for destructive buttons or using the same class for both buttons.

```js
customClass: {
    cancelButton: "stdbtn",
    confirmButton: "stdbtn",
},
confirmButtonColor: "#d33",
// with background changed to #484b51
```

3. ![image](https://user-images.githubusercontent.com/10560038/166286452-4e1b80eb-120a-4112-89c7-5daecc3b069b.png)
![image](https://user-images.githubusercontent.com/10560038/166286738-9e5aae48-7bca-47a8-a89c-aec781d7630a.png)

Reply:
Good to know that we can assign custom classes to the buttons 👍  
I'm going to merge things as-is for now, and I'll go through the CSS to try and harmonize it all a bit later. (I need to re-review the toasts as well but can't check the frontend atm)  
Thanks for your work!